app/src/views/GrantRound.vue

```diff
@@ -207,7 +207,7 @@ function useGrantRoundDetail() {
   const form = computed<{ token: string; amount: string }>(() => {
     return {
       token: grantRound.value.matchingToken.address,
-      amount: String(grantRound.value.minContribution),
+      amount: '0',
```

Comment:
@gdixon @thelostone-mc Can you verify this change is ok / what we want?

Reply:
Yup this is right